starter-code/index.js

```diff
@@ -1,10 +1,36 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
-
+var SortedList = function() {
+  this.items = [];
+  this.length = 0;
+};
+SortedList.prototype.add     = function(item) {
+  this.items.push(item);
+  this.items.sort(function(a, b) {
+    return a - b;
+  });
+  this.length = this.items.length;
+}
+SortedList.prototype.get     = function(pos) {
+  return this.items[pos - 1];
+}
+SortedList.prototype.max     = function() {
+  return Math.max.apply(null, this.items);
```

Comment:
Since we are working with a sorted array, does that give you some other idea how to do this? :)